src/DataGrid.tsx

```diff
@@ -504,6 +490,11 @@ function DataGrid<R, SR, K extends Key>(
   }
 
   function handleKeyDown(event: React.KeyboardEvent<HTMLDivElement>) {
```

Comment:
`handleKeyDown` is used twice now, is that alright?

Reply:
No, this needs to be fixed. I have a fix and I will push it soon.